### PR TITLE
Use Rollbar for server-side error tracking

### PR DESCRIPTION
**Pull request by alexsoble**
# Notes 

Talked online with @kevinrobinson and @jhilde, consensus is that setting up Rollbar for server-side error tracking is a good first step. This PR adds the code to set that up; I added the required ROLLBAR_ACCESS_TOKEN config to our production Heroku app already. 

Let's see how this does in the wild for a couple of weeks or months. My prediction is that it will be a significant improvement over Logentries. The next step in my mind would be some kind of SMS alerts to the team for critical errors like LDAP login errors.
 
# Issue 

+ #1205

### CODE REVIEW

**Comment by alexsoble** [comment]
Want to review @jhilde / @kevinrobinson or should this be a selfie?

File: config/initializers/rollbar.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  # config.use_sidekiq
+  # You can supply custom Sidekiq options:
+  # config.use_sidekiq 'queue' => 'default'
+
+  # If you run your staging application instance in production environment then
+  # you'll want to override the environment reported by `Rails.env` with an
+  # environment variable like this: `ROLLBAR_ENV=staging`. This is a recommended
+  # setup for Heroku. See:
+  # https://devcenter.heroku.com/articles/deploying-to-a-custom-rails-environment
+  config.environment = ENV['ROLLBAR_ENV'] || Rails.env
```
Comment (kevinrobinson) [inline config/initializers/rollbar.rb]: Is it worth adding the DEPLOYMENT_KEY or whatever else we do for UI analytics?

File: config/initializers/rollbar.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+
+  # Here we'll disable in 'test':
+  if Rails.env.test?
+    config.enabled = false
+  end
```
Comment (kevinrobinson) [inline config/initializers/rollbar.rb]: Should this be disabled in development mode too?

File: config/initializers/rollbar.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```
Comment (kevinrobinson) [inline config/initializers/rollbar.rb]: Will this come out in a different bucket of errors in the Rollbar UI, or same as JS?  I suspect it'll be obvious which one it's coming from just never tried it before! :)

**Comment by kevinrobinson** [comment]
@alexsoble Yeah I'd say 🚢 and just verify after or make some intentional errors and see how it works (notification levels, etc. too).

File: config/initializers/rollbar.rb (hunk 2 above)
Comment (alexsoble) [reply config/initializers/rollbar.rb]: Yes great catch!

File: config/initializers/rollbar.rb (hunk 3 above)
Comment (alexsoble) [reply config/initializers/rollbar.rb]: Yeah we'll see!

File: config/initializers/rollbar.rb (hunk 1 above)
Comment (alexsoble) [reply config/initializers/rollbar.rb]: Added using their `custom_data_method` thing.

**Pull request merged**